includes/transformer/class-post.php

```diff
@@ -975,22 +978,31 @@ private function get_quote_policy() {
 				return array( 'automaticApproval' => get_rest_url_by_path( sprintf( 'actors/%d/followers', $this->item->post_author ) ) );
 
 			case ACTIVITYPUB_INTERACTION_POLICY_ME:
-				return array( 'automaticApproval' => $this->get_me_actors() );
+				return array( 'automaticApproval' => $this->get_policy_for_me() );
 
 			default:
-				return array(
-					'automaticApproval' => 'https://www.w3.org/ns/activitystreams#Public',
-					'always'            => 'https://www.w3.org/ns/activitystreams#Public',
-				);
+				return $this->get_policy_for_anyone();
 		}
 	}
 
 	/**
-	 * Get the actor ID(s) for the `me` audience in the Quote interaction policy.
+	 * Get the public interaction policy.
+	 *
+	 * @return array The public interaction policy.
+	 */
+	private function get_policy_for_anyone() {
```

Comment:
```suggestion
	private function get_public_interaction_policy() {
```
Docs already say "Get the public interaction policy" and `The public interaction policy.`, I think this would also be a bit more elegant?

Reply:
Fixed with https://github.com/Automattic/wordpress-activitypub/pull/2221/commits/e3760c8cd967f4976b401dc991f6006a560dbe08